src/test/java/harness/utils/WorkspaceUtils.java

```diff
@@ -0,0 +1,48 @@
+package harness.utils;
+
+import bio.terra.cli.serialization.userfacing.UFWorkspace;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import harness.CRLJanitor;
+import harness.TestCommand;
+import harness.TestUser;
+
+/** Utilities for working with workspaces in CLI tests. */
+public class WorkspaceUtils {
+
+  /**
+   * Create a new workspace and register it with Janitor if this test is running in an environment
+   * where Janitor is enabled.
+   *
+   * @param workspaceCreator The user who owns the workspace. This user will be impersonated to in
+   *     the WSM workspaceDelete request.
+   */
+  public static UFWorkspace createWorkspace(TestUser workspaceCreator)
+      throws JsonProcessingException {
+    // `terra workspace create --format=json`
+    UFWorkspace workspace =
+        TestCommand.runAndParseCommandExpectSuccess(UFWorkspace.class, "workspace", "create");
+    CRLJanitor.registerWorkspaceForCleanup(workspace, workspaceCreator);
+    return workspace;
+  }
+
+  /**
+   * Create a new workspace and register it with Janitor if this test is running in an environment
+   * where Janitor is enabled.
+   *
+   * @param workspaceCreator The user who owns the workspace. This user will be impersonated to in
+   *     the WSM workspaceDelete request.
+   */
+  public static UFWorkspace createWorkspace(
```

Comment:
maybe name these `createTrackedWorkspace` so folks know they're special. I wouldn't have associated these calls with Janitor since the class is just `WorkspaceUtils`.

Reply:
Both names seem reasonable but I'd lean towards keeping this short - from a test reader perspective it's not important that the test objects are cleaned up later, it's the `createWorkspace` behavior I care about.

Added a comment about leaking workspaces if tests don't use this method